Gemfile.lock

```diff
@@ -2,74 +2,77 @@ PATH
   remote: .
   specs:
     show_for (0.6.0)
-      actionpack (>= 4.1, < 5.2)
-      activemodel (>= 4.1, < 5.2)
+      actionpack (>= 4.1, < 6.0)
```

Comment:
Maybe, it worth remove the upper bound like we did on Simple Form. See: https://github.com/plataformatec/simple_form/pull/1543#discussion-diff-180571824R23

Reply:
Sure thing 👍